File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = {}
        freq = [[] for i in range(len(nums) + 1)]

        for a in nums:
            count[a] = 1 + count.get(a, 0)
        for a, c in count.items():
            freq[c].append(a)

        res = []
        for i in range(len(freq) -1, 0, -1):
            for n in freq[i]:
                res.append(n)
                if len(res) == k:
                    return res

File: Data Structures & Algorithms/sort-an-array/submission-0.swift
class Solution {
    func sortArray(_ nums: [Int]) -> [Int] {
        var res = Array(nums)

        for i in 0..<res.count {
            for j in (i+1)..<res.count {
                if res[i] >= res[j] {
                    var temp = res[i]
                    res[i] = res[j]
                    res[j] = temp
                }
            }
        }

        return res
    }
}
